imap/message_test.c

```diff
@@ -307,7 +307,7 @@ static int usage(const char *name)
     exit(EX_USAGE);
 }
 
-void fatal(const char* s, int code)
+EXPORTED void fatal(const char* s, int code)
```

Comment:
If all fatal() are explicitly EXPORTED, can CFLAG_VISIBILITY be set in AM_LDFLAGS (globally for all targets) instead per target?

Reply:
Yep (either AM_LDFLAGS or AM_CFLAGS/AM_CXXFLAGS), and I plan to do something like this for master-only once 3.4 is released.

Reply:
Looking forward to see this in action.  When I did it in 19933ff48200387b the only problem was, as far as I remember, that the `fatal()` were not exported from the executables to the libraries.

Reply:
> Yep (either AM_LDFLAGS or AM_CFLAGS/AM_CXXFLAGS), and I plan to do something like this for master-only once 3.4 is released.

Now 3.4 is released.